backbone/src/main/java/org/researchstack/backbone/answerformat/GenderAnswerFormat.java

```diff
@@ -0,0 +1,48 @@
+package org.researchstack.backbone.answerformat;
+
+import android.content.Context;
+
+import org.researchstack.backbone.R;
+import org.researchstack.backbone.model.Choice;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by TheMDP on 1/4/17.
+ */
+
+public class GenderAnswerFormat extends ChoiceAnswerFormat {
+
+    public GenderAnswerFormat(Context context) {
+        super(ChoiceAnswerStyle.SingleChoice, createChoices(context));
+    }
+
+    /**
+     * @param context used to build strings for male, female, other
+     * @return Choice array for gender
+     */
+    static Choice[] createChoices(Context context) {
```

Comment:
I'm not sure what you mean.  Do you mean like a new constructor in ChoiceAnswerFormat?  
For more context, I made this method a static method to avoid getting each String twice from Context when making the Choice objects.